modules/rostests/kmtests/include/kmt_test.h

```diff
@@ -169,9 +169,9 @@ VOID KmtFinishThread(IN PKTHREAD Thread OPTIONAL, IN PKEVENT Event OPTIONAL);
 #elif defined KMT_USER_MODE
 DWORD KmtRunKernelTest(IN PCSTR TestName);
 
-VOID KmtLoadDriver(IN PCWSTR ServiceName, IN BOOLEAN RestartIfRunning);
+DWORD KmtLoadDriver(IN PCWSTR ServiceName, IN BOOLEAN RestartIfRunning);
```

Comment:
At this point we might want a `KmtLoadAndOpenDriver` helper to lighten up the calling code.

Reply:
Added it.